usaspending_api/api_contracts/contracts/subawards/Subawards.md

```diff
@@ -0,0 +1,51 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Subawards
+
+These endpoints are used to power USAspending.gov's subaward listings.
+
+# Subaward List
+
+This endpoint returns a filtered set of subawards.
+
+## Subawards [/api/v2/subawards/]
+
+This endpoint returns a list of data that is associated with the award profile page.
+
+### Subawards [POST]
+
++ Request (application/json)
+    + Attributes (object)
+        + page: 1 (required, number)
+        + limit: 10 (optional, number)
+        + filters (required, FilterObject)
+        + order: (required, enum)
+            + asc
+            + desc 
+            
++ Response 200 (application/json)
+    + Attributes
+        + `limit` (optional, number)
+        + `results`(array[SubawardResponse])
```

Comment:
`results` should be: (required, array[SubawardResponse], fixed-type).
Are the other responses not required?

Reply:
Not on the advanced search page, which I based it off of.